apps/website/docs/react/api/make-styles.md

```diff
@@ -213,3 +213,29 @@ const useClasses = makeStyles({
   },
 });
 ```
+
+### CSS Fallback Properties
+
+Griffel supports CSS fallback properties in order to support older browsers.
+
+Any CSS property accepts an array of values which are all added to the styles.
+Every browser will use the latest valid value (which might be a different one in different browsers, based on supported CSS in that browser): 
+
+```js
+import { makeStyles } from '@griffel/react';
+
+const useClasses = makeStyles({
+  root: {
+    overflowY: ['scroll', 'overlay'],
```

Comment:
Ironically, csstype typings currently do not accept `overlay` as a valid value for `overflow` 😢

Reply:
Yup, https://github.com/frenic/csstype/issues/62